app/models/server/raw/Rooms.js

```diff
@@ -109,15 +109,16 @@ export class RoomsRaw extends BaseRaw {
 		return this.find(query, options);
 	}
 
-	findByTeamId(teamId, options = {}) {
-		const query = {
+	findByTeamId(teamId, options = {}, query = {}) {
+		const myQuery = {
 			teamId,
+			...query,
```

Comment:
please move `...query` one line up, so we can make sure the `query` doesn't override our `teamId` filter